honestcash-v2/src/app/main/components/story-author/story-author.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="row mb-4">
```

Comment:
some of these code is duplicated among other story- related components. I'll also refactor them